buildenv

```diff
@@ -0,0 +1,178 @@
+# Update bump details accordingly. Use bump check to confirm.
+# bump: readline-version /READLINE_VERSION="(.*)"/ https://ftp.gnu.org/gnu/readline/readline-8.3-rc1.tar.gz|semver:*
+# READLINE_VERSION="V.R.M" # Specify a stable release
+# export ZOPEN_STABLE_TAG="v${READLINE_VERSION}"
+
+###
+### Build control file for the readline port
+###
+
+###
+### Required Environment Variables
+###
+
+## Specify the default build line, either 'DEV' or 'STABLE' (default: STABLE)
+export ZOPEN_BUILD_LINE="STABLE"
+
+## Required IF ZOPEN_BUILD_LINE='STABLE'
+export ZOPEN_STABLE_URL="https://ftp.gnu.org/gnu/readline/readline-8.3-rc1.tar.gz"  # Specify the stable build URL (either git or tarball)
```

Comment:
Can you set the variable above: https://github.com/zopencommunity/readlineport/pull/1/files#diff-fa4eb8fc4019dae6d59ab4aa69f58f285bdf7309cbea95235e60cfa742bf1368R3 and also the bump line.
The bump automation is important for detecting upstream release updates